data-acquisition/nomask_generator/No_mask_downloader.ipynb

```diff
@@ -0,0 +1,170 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Downloading and Generating Masks for Non Class Examples"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 45,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from PIL import Image\n",
+    "from google_images_download import google_images_download\n",
+    "import Augmentor\n",
+    "import glob\n",
+    "import shutil\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 46,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def create_masks(path):\n",
+    "    \"\"\"\n",
+    "    This function generate empty black masks for each of \n",
+    "    the images downloaded from Google. \n",
+    "    \"\"\"\n",
+    "    x = path.rstrip('\\n')\n",
+    "    t = x.split('/')[-1].rstrip('.jpg')\n",
+    "    im = Image.open(x)\n",
+    "    im_size = im.size\n",
+    "    img = Image.new('RGB', im_size)\n",
+    "    img.save(\"no_mask/masks/\"+\"mask_\"+t+\".jpg\", \"JPEG\")\n",
+    "    return"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 49,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def create_image_dataset(search_term, number_of_images, augmentation=0):\n",
```

Comment:
Can we use boolean variables (`True/False`) instead of 0/1 for flags?

Reply:
That's actually not a boolean, it's an integer specifying how many augmentations should be generated.

Reply:
Ah I see, my bad, two follow up points
1. Can we rename the argument to indicate its a count of the number of augmented images generated?
2. Why do we need to generate the augmented images? I believe it is good practice to generate these on the fly during training/testing (@pshivraj)

Reply:
I think its good to add augmentation at one place to have a better sense as to what's happening. Also I would recommend using scripts which are importable rather notebooks for better readability and version control.